client/Windows/wf_event.c

```diff
@@ -130,15 +130,16 @@ LRESULT CALLBACK wf_ll_kbd_proc(int nCode, WPARAM wParam, LPARAM lParam)
 				DEBUG_KBD("keydown %d scanCode 0x%08lX flags 0x%08lX vkCode 0x%08lX",
 				          (wParam == WM_KEYDOWN), p->scanCode, p->flags, p->vkCode);
 
-				if (wfc->fullscreen_toggle &&
-				    ((p->vkCode == VK_RETURN) || (p->vkCode == VK_CANCEL)) &&
-				    (GetAsyncKeyState(VK_CONTROL) & 0x8000) &&
-				    (GetAsyncKeyState(VK_MENU) & 0x8000)) /* could also use flags & LLKHF_ALTDOWN */
+
+				if (wfc->fullscreen_toggle && (p->vkCode == VK_RETURN || p->vkCode == VK_CANCEL))
 				{
-					if (wParam == WM_KEYDOWN)
+					if (keystates[0x38] && keystates[0x1D]) // left ALT, left CTRL
```

Comment:
2 remarks (sorry, did not publish that one yesterday):
1. Use the constants to address the keys (e.g. `VK_CANCEL`) instead of numbers
2. Check for `VK_CONTROL`, `VK_LCONTROL`, `VK_RCONTROL` (not every keyboard has both control keys)
3. Also check for `VK_MENU`, `VK_LMENU` and `VK_RMENU` (the `ALT` keys)

Reply:
or better yet, I´ll push these changes to your branch ;)